src/App.js

```diff
@@ -1,73 +1,83 @@
-import {LandingPage} from "Pages/LandingPage/LandingPage";
+import { LandingPage } from "Pages/LandingPage/LandingPage";
 import Mockman from "mockman-js";
-import { ToastContainer, } from 'react-toastify';
-import {HomePage} from "Pages/HomePage/HomePage";
-import {ProfilePage} from "Pages/ProfilePage/ProfilePage";
-import {Page404} from "Pages/Page-404/Page404";
-import {Route,Routes,Outlet, useLocation} from "react-router-dom";
+import { ToastContainer } from "react-toastify";
+import { HomePage } from "Pages/HomePage/HomePage";
+import { ProfilePage } from "Pages/ProfilePage/ProfilePage";
+import { Page404 } from "Pages/Page-404/Page404";
```

Comment:
Instead of multiple import statement you can create a single index.js file for `pages` folder and can import that here in just one statement like this
`import {HomePage, ProfilePage, Page404} from "./pages";`

Reply:
I will do this Thanks for reviewing the PR